pkg/prometheus/rules_test.go

```diff
@@ -160,10 +161,33 @@ func shouldSplitUpLargeSmallIntoTwo(t *testing.T) {
 	}
 
 	if len(configMaps) != 2 {
-		t.Fatalf("expected rule files to be split up into two ConfigMaps, but got '%v' instead", len(configMaps))
+		t.Fatalf("expected rule files to be split up into three ConfigMaps, but got '%v' instead", len(configMaps))
```

Comment:
Same as previous comment
```suggestion
		t.Fatalf("expected rule files to be split up into two ConfigMaps, but got '%v' instead", len(configMaps))
```